build.sbt

```diff
@@ -80,3 +94,21 @@ fork in Test := true
 
 // forward stdin/out to fork, so the OpenIE CLI can be run in sbt.
 connectInput in run := true
+
+// The OpenIE codebase has many insignificant style warnings that appear
+// whenever the code is compiled. There are also some important-sounding
+// warnings which don't seem to affect the behavior but should be
+// fixed eventually:
+//
+//   Classes must override both equals and hashCode, not just one
+//   Line is more than 100 characters long
+//   Tab in Scala file
+//   Unusual class name; consider renaming
+//   illegal.imports.message
```

Comment:
Hrm I thought it might be possible to override the ScalastyleConfiguration from within this project's `build.sbt`, but it looks like you're right and you'd need to modify the style plugin to get that functionality (which could be useful, but probably isn't worth the time right now).

If it's quick to replace the tab characters with \t in the regex strings I think it'd be good to add in here, since it'll kill that warning and also help with the regex readability.

`scala.collection.JavaConversions._` contains a bunch of implicit conversions between conceptually equivalent Java and Scala collection types.  It's generally advised against because implicit conversions make the code much more difficult to reason about.  The better import is `scala.collection.JavaConverters._`, which through *different* implicit magic adds the `.asJava` method to Scala collections and the `.asScala` method to Java collections.  Ideally we'd switch the imports and then go through and add the explicit conversion calls, but that might be more work than we want to do on this right now.  If so, we probably don't want to hide that warning, since the conversions are considered a dangerous feature.

Understood on your points about the class names and hashCode/equals, but I think we'd want to let those warnings keep printing.  hashCode/equals is a legitimate problem that somebody should eventually fix, and the weird class names could hurt code readability when you see `stringFormat` and expect it to be a value or method but instead it's a class.